src/Core/System/User/Command/UserChangePasswordCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\Core\System\User\Command;
+
+use Shopware\Core\Framework\Adapter\Console\ShopwareStyle;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepository;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsFilter;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+
+class UserChangePasswordCommand extends Command
+{
+    protected static $defaultName = 'user:change-password';
+
+    /**
+     * @var EntityRepository
+     */
+    private $userRepository;
+
+    public function __construct(EntityRepository $userRepository)
+    {
+        $this->userRepository = $userRepository;
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('username', InputArgument::REQUIRED, 'Username of the user')
+            ->addOption('password', 'p', InputOption::VALUE_REQUIRED, 'New password for the user');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
should we ask for the current password first?

Reply:
As this is a CLI command, I would not ask the current password. This command can only be run by users who have access to it anyway. This is a quick way to create a new password for an admin user when the password was forgotten.